server/lobbyconnection.py

```diff
@@ -895,66 +951,121 @@ def _set_user_agent_and_version(self, user_agent, version):
 
         async with self._db.acquire() as conn:
             if type == "start":
-                result = await conn.execute("SELECT uid, name, version, author, ui, date, downloads, likes, played, description, filename, icon FROM table_mod ORDER BY likes DESC LIMIT 100")
+                result = await conn.execute(
+                    select([
+                        table_mod
```

Comment:
This really doesn't need it's own line :P

Reply:
I grudgingly agree :P